tests/unit/sagemaker/serve/constants.py

```diff
@@ -11,3 +11,7 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 from __future__ import absolute_import
+
+
+MOCK_IMAGE_CONFIG = {"RepositoryAccessMode": "Vpc"}
```

Comment:
Are both these fields required?

Reply:
No, as suggested by type hint, they're all optional and default to None

Reply:
Validation is handled by the SM Create Model API - 
(Internal Code links: https://tiny.amazon.com/6sx1sxix, https://tiny.amazon.com/xulosxe4) 
The validation can be forfeited at this point - but model creation will fail if the conditions aren't met - and any errors should be surfaced to cx.